excel-connector-test/pom.xml

```diff
@@ -33,5 +33,16 @@
         </configuration>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-deploy-plugin</artifactId>
+          <version>3.0.0-M1</version>
+          <configuration>
+            <skip>true</skip>
```

Comment:
there is no need to deploy the testproject to maven artifactory
I'll explain that in the how to guide.